modules/editor/project.el

```diff
@@ -22,12 +22,13 @@
     "Convert FILENAME to be relative to the root of a `project.el' PROJECT.  Defaults to `project-current'"
     (file-relative-name filename (project-root (or project (project-current))))))
 
+(elpaca transient) ;; Make sure we're using an up-to-date copy of transient?
 (use-package disproject
-  :straight
-  '(disproject
-    :type git
-    :host github
-    :repo "aurtzy/disproject")
+  :ensure
+    '(disproject
```

Comment:
Do you prefer extra indentation for things like this?

Reply:
I'd like it if that was on the same line.

Do these lists have to be quoted? That's kind of annoying if so.

Reply:
disproject did not need to be quoted, and fixed the indendentation in 6e2b733. Thanks!